srtcore/core.cpp

```diff
@@ -7325,6 +7325,70 @@ void CUDT::updateAfterSrtHandshake(int srt_cmd, int hsv)
     }
 }
 
+
+int CUDT::packLostData(CPacket& packet, uint64_t &origintime)
+{
+    while (0 <= (packet.m_iSeqNo = m_pSndLossList->getLostSeq()))
```

Comment:
Avoid inversion.

Reply:
BTW is 0 a valid value?

Reply:
Yes. -1 is invalid due to the most significant bit in the SEQNO field in the header to distinguish control and data packet. Maximum value is `0x7FFFFFFF`.